src/test/java/seedu/address/model/person/ExperienceTest.java

```diff
@@ -27,15 +27,15 @@ public void isValidExperience() {
         // invalid experience
```

Comment:
As mentioned earlier, you should add adding a long value as an invalid args test case!